docs/usage/configuration.md

```diff
@@ -40,20 +40,39 @@ nextflow run nextflow-io/hello -c nextflow.config
 
 For a complete list and detailed descriptions of all available configuration parameters, please refer to the [Parameters](./parameters.md) section.
 
-## How the plugin determines the carbon intensity (ci) value
+## Carbon intensity (ci)
+
+### How are they determined by default?
 
 The plugin uses the `ci`, `location`, and `apiKey` parameters to determine the carbon intensity (in gCO₂eq/kWh) that is used in the energy impact calculations. The logic is as follows:
 
 1. **If `ci` is explicitly set**, this value is used directly as the carbon intensity, and no API call is made.
 2. **If `ci` is not set**, but both `location` and `apiKey` are provided, the plugin will query the [Electricity Maps API](https://www.electricitymaps.com/) for a real-time carbon intensity value for the specified zone. The API call is made once per Nextflow task to retrieve the most up-to-date carbon intensity.
 3. **If only `location` is set**, the plugin will fallback to a default value for the specified zone. 
-3. **If neither `ci` nor valid `location` and `apiKey` are provided**, the plugin will  fallback to a global default value.
+4. **If neither `ci` nor valid `location` and `apiKey` are provided**, the plugin will  fallback to a global default value.
 
 > Carbon intensity data is retrieved from [Electricity Maps](https://www.electricitymaps.com/) and used under the [Open Database License (ODbL)](https://opendatacommons.org/licenses/odbl/1-0/). See the full attribution and license terms [here](https://nextflow-io.github.io/nf-co2footprint/).
 
+### Accounting for a personal energy mix
```

Comment:
I think we could simply use the term `marked-based CI` here. From what I understood, own energy production through solar panels would only influence the amount of energy consumed with the `location-based` CI (which wouldn't help to differentiate in this context). On the other hand, I think it can be included in the `market-based` CI calculation. Still didn't find reliable sources though.
 
But then, we could rename it to `ciMarket`, and mention an example with an green energy tariff.

In the description of the normal `ci`, we should then describe that this corresponds to a location-based CI, and maybe briefly describe here the difference of the two.